src/baselaplace.jl

```diff
@@ -19,8 +20,8 @@ Retrieves the desired (sub)set of model parameters and stores them in a list.
 
 ```julia-repl
 using Flux, LaplaceRedux
-nn = Chain(Dense(2,1))
-la = Laplace(nn)
+nn = Chain(Dense(2,1))                                      # defines a neural network with one hidden layer that takes a two-dimensional input 
+la = Laplace(nn)                                            # and produces a one-dimensional output
 LaplaceRedux.get_params(la)
```

Comment:
IMO having the comments on separate lines would make more sense, since you sometimes do not have a correspondence between the comment and the line

Reply:
This is a totally minor issue, so feel free to ignore

Reply:
Fixed.